pgbot/__init__.py

```diff
@@ -1,74 +1,109 @@
             prim = "| Primary Guild"
-            common.guild = server
+            common.guild = guild
 
-        print(" -", server.name, "| Number of channels:", len(server.channels), prim)
+        print(" -", guild.name, "| Number of channels:", len(guild.channels), prim)
         if common.GENERIC:
             continue
 
-        for channel in server.channels:
-            if channel.id == common.ServerConstants.DB_CHANNEL_ID:
-                common.db_channel = channel
-                await db.init()
-            elif channel.id == common.ServerConstants.LOG_CHANNEL_ID:
+        for channel in guild.channels:
+            if channel.id == common.GuildConstants.DB_CHANNEL_ID:
+                if not common.TEST_MODE:
+                    snakecore.config.conf.db_channel = common.db_channel = channel
+                await snakecore.db.init_discord_db()
+            elif channel.id == common.GuildConstants.LOG_CHANNEL_ID:
                 common.log_channel = channel
-            elif channel.id == common.ServerConstants.ARRIVALS_CHANNEL_ID:
+            elif channel.id == common.GuildConstants.ARRIVALS_CHANNEL_ID:
                 common.arrivals_channel = channel
-            elif channel.id == common.ServerConstants.GUIDE_CHANNEL_ID:
+            elif channel.id == common.GuildConstants.GUIDE_CHANNEL_ID:
                 common.guide_channel = channel
-            elif channel.id == common.ServerConstants.ROLES_CHANNEL_ID:
+            elif channel.id == common.GuildConstants.ROLES_CHANNEL_ID:
                 common.roles_channel = channel
-            elif channel.id == common.ServerConstants.ENTRIES_DISCUSSION_CHANNEL_ID:
+            elif channel.id == common.GuildConstants.ENTRY_CHANNEL_IDS["discussion"]:
                 common.entries_discussion_channel = channel
-            elif channel.id == common.ServerConstants.CONSOLE_CHANNEL_ID:
+            elif channel.id == common.GuildConstants.CONSOLE_CHANNEL_ID:
                 common.console_channel = channel
-            elif channel.id == common.ServerConstants.RULES_CHANNEL_ID:
+            elif channel.id == common.GuildConstants.RULES_CHANNEL_ID:
                 common.rules_channel = channel
-            for key, value in common.ServerConstants.ENTRY_CHANNEL_IDS.items():
+            for key, value in common.GuildConstants.ENTRY_CHANNEL_IDS.items():
                 if channel.id == value:
                     common.entry_channels[key] = channel
 
+    await common.bot.load_extension("pgbot.exts.core_commands.help")
+    await common.bot.load_extension("pgbot.exts.core_commands.admin")
+    await common.bot.load_extension("pgbot.exts.core_commands.user")
+
+    async with snakecore.db.DiscordDB("blacklist", list) as db_obj:  # disable blacklisted commands
+        for cmd_qualname in db_obj.obj:
+            cmd = common.bot.get_command(cmd_qualname)
+            if cmd is not None:
+                cmd.update(enabled=False)
```

Comment:
I think this was discussed before (but I forgot whether we reached a resolution at it)... but this being here means that we can't actively block a command without forcing a reload. It's not a big deal, but it's something to keep in mind

Reply:
That discussion was at https://github.com/pygame-community/PygameCommunityBot/pull/380#discussion_r852565270 .

In any case, no full bot restart would occur through the `.update()` method as that is not what the method is *written* to do. The only thing here that is reloaded is the command itself. However I now see that things could be more efficient here, so I should probably do `.enabled = False` instead.
```suggestion
                cmd.enabled = False
```